exercises/concept/ellens-alien-game/.docs/instructions.md

```diff
@@ -0,0 +1,116 @@
+# Instructions
+
+Ellen is making a game where the player has to fight aliens.
+She has just learned about Object Oriented Programming (OOP) and is eager to take advantage of what this paradigm has to offer.
+
+To Ellen's delight, you have offered to help and she has given you the task of programming the aliens that the player has to fight.
+
+## 1. Create the Alien Class
+
+Define the Alien class so that it accepts an x and a y position, and puts them into `x` and `y` variables.
+Every alien starts off with a health of 3, so initialize the `health` variable too.
+
+```python
+>>> alien = Alien(2, 0)
+>>> alien.x
+2
+>>> alien.y
+0
+>>> alien.health
+3
+```
+
+Now, each alien should be able to keep track of it's own position and health.
+
+## 2. The `hit` Method
+
+Ellen would like the Alien class to have a `hit` method that decrements the health of an alien object when it is called.
+This way, she can simply call the `hit` method instead of having to manually change an alien's health.
+
+For example:
+
+```python
+>>> alien = Alien(0, 0)
+>>> alien.health
+3 # same as before
+>>> alien.hit()
+>>> alien.health
+2
+```
+
+## 3. The `is_alive` Method
+
+You realize that if the health keeps decreasing, at some point it will probably hit 0 (or even less!).
+It would be a good idea to add an `is_alive` method that Ellen can quickly call to check if the alien is... well... alive.
+
+```python
+>>> alien.health
+1
+>>> alien.is_alive()
+True
+>>> alien.hit()
+>>> alien.health
+0
+>>> alien.is_alive()
+False
+```
+
+## 4. The `teleport` Method
+
+In Ellen's game, the aliens have the ability to teleport! 
+You will need to write a `teleport` function that takes new x and y positions, and changes the alien's position accordingly.
```

Comment:
Doh! 🤦  I called it a function instead of a method.  Also, same hesitation about the back-quotes.  Also (also), are the new x and y going to be `coordinate_x` and `coordinate_y` because before it was `x_coordinate` and `y_coordinate`?